faq.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>FAQ | 판다마켓</title>
+<!-- pretendard web-font: https://github.com/orioncactus/pretendard -->
+<link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css">
+<link rel="stylesheet" href="style.css">
+<link rel="icon" href="favicon.png" type="image/png">
+</head>
+
+<body>
+<div class="wrap">
+  <header id="header">
+    <div class="container header">
+      <h1 class="site-name">
+        <a href="/" class="root-link">
+          <img src="img/logo.png" alt="로고 이미지" class="logo">
+          <img src="img/site-name.png" alt="판다마켓">
+        </a>
+      </h1>
+      <a href="/login.html" class="btn">로그인</a>
+    </div>
+    <!-- /.container -->
+  </header>
+
+  <section class="section">
+    <div class="container">
+      <h2>FAQ</h2>
+    </div>
+    <!-- /.container -->
+  </section>
+  <!-- /.section -->
+
+  <div class="bottom-visual bg-sky">
+    <div class="container">
+      <h2 class="heading">믿을 수 있는<br>판다마켓 중고 거래</h2>
+    </div>
+    <!-- /.container -->
+  </div>
+  <!-- / bottom-visual -->
+
+  <footer id="footer">
+    <div class="container">
+      <p class="cr">&copy;codeit - 2024</p>
+      <ul class="links">
+        <li><a href="privacy.html">Privacy Policy</a></li>
+        <li><a href="faq.html">FAQ</a></li>
+      </ul>
+      <ul class="sns">
+        <li>
+          <a href="https://www.facebook.com/" target="_blank">
+            <img src="img/ico-facebook.png" alt="FaceBook">
+          </a>
+        </li>
+        <li>
+          <a href="https://x.com/" target="_blank">
+            <img src="img/ico-twitter.png" alt="X">
+          </a>
+        </li>
+        <li>
+          <a href="https://youtube.com/" target="_blank">
+            <img src="img/ico-youtube.png" alt="Youtube">
+          </a>
```

Comment:
```suggestion
          <a href="https://youtube.com/" target="_blank" rel="noopener noreferrer">
            <img src="img/ico-youtube.png" alt="Youtube">
          </a>
```
`target="_blank"` 를 사용하여 새 탭이나 새 창에서 링크를 열 때, 열린 새 페이지는 window.opener 객체를 통해 원래 페이지에 접근할 수 있습니다. 이는 잠재적인 보안 위험으로 아래처럼 rel="noopener"를 사용하여 window.opener 참조를 제거를 통해 원래 페이지에 접근할 수 없도록 설정합니다.